reslib/pom.xml

```diff
@@ -47,10 +47,10 @@
 									<goal>wget</goal>
 								</goals>
 								<configuration>
-									<url>https://github.com/liferay/liferay-portal/raw/0f75db5b5c6a8a7c6b07295cc12acdf362bfb1b4/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
+									<url>https://github.com/liferay/liferay-portal/raw/6.2.x/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
```

Comment:
Yep, as I mentioned that shouldn't happen and shouldn't had happened before. Every change in alloy zip should imply a naming change too. If not, probably it would be a mistake. Thanks!